tests.py

```diff
@@ -12,8 +12,43 @@
 
 RULES_PATH = os.path.abspath(os.path.join('.', 'sample_rules'))
 
+def format_ipc_input(data):
+    return [ (x.strip(), y.strip()) for x, y in \
+             [ elm.split('=') for elm in data.split('\n') ] ]
+
+def format_ipc_output(data):
+    return (('', '') if data == '' else tuple(data.strip().split(' = ')))
```

Comment:
I still don't like this, but it will probably change when the split between signals and logs is done, anyway, so I'm ok with keeping it.